kdc/bx509d.c

```diff
@@ -1855,6 +1858,7 @@ authorize_TGT_REQ(struct bx509_request_desc *r)
         ret = kdc_authorize_csr(r->context, "get-tgt", r->req, p);
     krb5_free_principal(r->context, p);
     hx509_request_free(&r->req);
+    r->req = NULL;
```

Comment:
Per Nico this is unnecessary, but harmless, since hx509_request_free nulls out req.